src/components/views/Profile.tsx

```diff
@@ -112,6 +113,12 @@ const Profile = () => {
         return null;
     };
 
+    useEffect(() => {
+        if(userData){
+            console.log(userData);
+            console.log(userData.achievements)
```

Comment:
after you're done with the branch, remember to remove the console.logs

Reply:
Yeah, was planning to add more profile pics, at least as many as there achievements rn, then last check because I'm paranoid and then I'll remove the console logs